hls-test-utils/src/Test/Hls/Util.hs

```diff
@@ -112,10 +112,13 @@ data GhcVersion
   | GHC88
   | GHC86
   | GHC84
+  | GHC901
```

Comment:
Hi, dont want to make you add more changes but i am preparing a pr which promotes those data types to ghcide to being able to use them in all the codebase. In that pr i am using `GHC90` to be consistent with the rest of constructors: https://github.com/jneira/haskell-language-server/pull/24/files#diff-a54f7055c79019e6a0bfbac7b8dcce0048de756ad4ea47d02a161177dd57d9acR593-R611

Will change this to match my changes after this pr so dont worry if you dont want/have time to make the GHC901 -> GHC90 change.

Reply:
OK. As far as i know there are many changes since ghc 9.1, so I just used `GHC901` here, though I'm not sure if the incoming changes will break something.

Reply:
ghc 9.1 will have its own `GHC91` and we could change cpp conditions to make `GHC90` and `GHC91` incompatible (with `MIN_VERSION_GLASGOW_HASKELL(9,0,0,0) && !MIN_VERSION_GLASGOW_HASKELL(9,1,0,0)`) so it will throw a runtime (i am afraid) error